### PR TITLE
Model Serialization Fix

### PR DESCRIPTION
**Pull request by benjamin-pilgrim**
Ported model serialization fix from my implementation.

### CODE REVIEW

**Comment by chrissm79** [comment]
@TGNThump PRs looks amazing, thank you!! I probably won't be able to dig into them and test until this afternoon but from first glance I'll likely merge them all in. 👍

File: src/WebSocket/TransportManager.php [hunk 1]
```diff
@@ -238,13 +243,22 @@ public function broadcast(array $payload)
...
 
         if (! $field || empty($subscribers)) {
             return;
         }
 
-        $event = $field->transform(array_get($payload, 'event'));
+        $event = array_get($payload, 'event');
+
+        foreach ($event as $key => $value) {
+            if (is_array($value) && array_keys($value) == ["class", "id", "relations", "connection"]){
```
Comment (chrissm79) [inline src/WebSocket/TransportManager.php:254]: @TGNThump When trying to test this out, the `class`, `relations` and `connection` keys are missing on the `$payload` array passed into the `RedisBroadcaster`... do I need to do anything in the `Event` class that was fired to get this to serialize properly or are you doing something manual?

Comment (benjamin-pilgrim) [reply src/WebSocket/TransportManager.php:254]: @chrissm79 The modified version of the SerializesModels trait (Nuwave\Lighthouse\Subscriptions\SerializesModels) needs to be on the event. I couldn't think of any other way to do this automatically without screwing with Laravel core code, so if you have any ideas, feel free to change it.

**Comment by chrissm79** [comment]
@TGNThump Thanks for your work on this one!!!

**Pull request merged**